pyproject.toml

```diff
@@ -45,7 +45,7 @@ dependencies = [
   "dask",
   "netCDF4",
   "numpy >= 1.12",
-  "pandas",
+  "pandas < 2.1.1",
```

Comment:
Won't this affect people's local distributions?  If we don't want that, would it be better to add a limit to the yamls?

Reply:
This will. I'm open to hacking the yamls.